filebeat/input/udp/config.go

```diff
@@ -3,6 +3,8 @@ package udp
 import (
 	"time"
 
+	humanize "github.com/dustin/go-humanize"
```

Comment:
I might be wrong here, but AFAIK the prefix "go-" is removed automatically. So you don't need to alias this import.

Reply:
I really need to fix my editor to stop adding theses.

Reply:
@kvch Your vim doesn't do that?